src/patternfly/components/Breadcrumb/examples/breadcrumb-with-title-example.hbs

```diff
@@ -0,0 +1,34 @@
+{{#> breadcrumb breadcrumb--modifier="pf-u-my-md"}}
+  {{#> breadcrumb-list}}
+    {{#> breadcrumb-item}}
+      {{#> breadcrumb-link breadcrumb-link--href="#"}}
+        Section Home
+      {{/breadcrumb-link}}
+    {{/breadcrumb-item}}
+    {{#> breadcrumb-item}}
+      {{#> breadcrumb-link breadcrumb-link--href="#"}}
+        Section Title
+      {{/breadcrumb-link}}
+    {{/breadcrumb-item}}
+    {{#> breadcrumb-item}}
+      {{#> breadcrumb-link breadcrumb-link--href="#"}}
+        Section Title
+      {{/breadcrumb-link}}
+    {{/breadcrumb-item}}
+    {{#> breadcrumb-item}}
+      {{#> breadcrumb-link breadcrumb-link--href="#"}}
+        Section Title
+      {{/breadcrumb-link}}
+    {{/breadcrumb-item}}
+    {{#> breadcrumb-item}}
+      {{#> breadcrumb-link breadcrumb-link--href="#"}}
+        Section Title
+      {{/breadcrumb-link}}
+    {{/breadcrumb-item}}
+    {{#> breadcrumb-item breadcrumb-item--current="true"
+      breadcrumb-item--modifier="pf-m-title"
+      breadcrumb-item--attribute='title="Main Title"'}}
```

Comment:
I ran this through VO and I'm not sure this is necessary. It announces the heading directly after the title text is announced so it seems redundant. Was this needed for JAWS or NVDA?